state/modelsummaries.go

```diff
@@ -311,7 +317,8 @@ func (p *modelSummaryProcessor) fillInMigration() error {
 		}},
 	})
 	pipe.Batch(100)
-	iter := pipe.Iter()
+	var iter mongo.Iterator = pipe.Iter()
+	defer iter.Close()
```

Comment:
why the need to cast it to a different type?

Reply:
Because pipe is an *mgo.Pipe, and returns an *mgo.Iter. Pipe usage is much more limited, didn't feel worthwhile going and making that an interface yet.